src/services/deckService.js

```diff
@@ -19,14 +19,65 @@ class DeckService {
     await sheet.loadCells("H4:H520");
     const cells = [];
     for (let i = 4; i < 100; i++) {
-      cells.push(sheet.getCellByA1(`H${i}`));
+      //cells.push(sheet.getCellByA1(`H${i}`));
+      if (!sheet.getCellByA1(`H${i}`)._rawData.formattedValue) {
+        console.log(i-1);
+        return i;
+      }
     }
-    console.log(sheet.cellStats);
-    await sheet
+    //console.log(sheet.cellStats); why do this?
+    await sheet; //???
   };
-}
 
+  checkDecks = async () => {
+    const sheet = this.doc.sheetsByIndex[0];
+
+    await sheet.loadCells("O4:O80");
+    await sheet.loadCells("X4:X80");
+    //console.log("cells loaded correctly");
+
+    //SHOWS DECKS IN an array in discordMessage with their respective number and meta%
+    let discordMessage = "";
+
+    for (let i = 4; i < 14; i++) {
+      const currentDeckCell = sheet.getCellByA1(`O${i}`)._rawData
+        .formattedValue;
+      const currentDeckPercentage = sheet.getCellByA1(`X${i}`)._rawData
+        .formattedValue;
+      if (!currentDeckCell) {
+        break;
+      }
+
+      discordMessage =
+        discordMessage +
+        (i -
+          3 +
+          "." +
+          " " +
+          currentDeckPercentage +
+          " " +
+          currentDeckCell +
+          "\n");
+    }
```

Comment:
Small tip:

You can build strings like this: ``discordMessage = `${i} - ${3}. ${currentDeckPercentage} ${currentDeckCell \n` ``